CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @DataDog/dd-trace-js
+/packages/dd-trace/src/appsec/ @DataDog/appsec-js
```

Comment:
Isn't all of `appsec-js` also in `dd-trace-js`?

Reply:
Actually no, check it out:
https://github.com/orgs/DataDog/teams/dd-trace-js/members
https://github.com/orgs/DataDog/teams/appsec-js/members
That could be changed I guess, but the point of the different codeowner is to not ping the whole dd-trace team when it's a change that's purely under the appsec folder.
I guess it's just a question on how we want to organize reviews etc..., I'm proposing this change to reduce the tracer team workload/noise